src/GitLink/Providers/VisualStudioTeamServicesProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using GitTools.Git;
+
+namespace GitLink.Providers
+{
+    public class VisualStudioTeamServicesProvider : ProviderBase
+    {
+        private readonly Regex _visualStudioTeamServicesRegex = new Regex(@"(?<url>(?<companyurl>(?:https://)?(?<accountname>([a-zA-Z0-9\-\.]*)?)\.visualstudio\.com/)(?<project>[a-zA-Z0-9\-\.]*)/?_git/(?<repo>[^/]+))");
+
+        public VisualStudioTeamServicesProvider() 
+            : base(new GitPreparer())
+        {
+        }
+
+        public override string RawGitUrl
+        {
+            get
+            {
+                return "";
```

Comment:
Please use string.Empty (easier for the brain to read, the brain now has to analyze empty string vs space).